test/test-execute/exec-personality-ppc64.service

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=Test for Personality=ppc64
+
+[Service]
+ExecStart=/bin/sh -c 'echo $(uname -m); exit $(test $(uname -m) = "ppc64" -o $(uname -m) = "ppc64le")'
+Type=oneshot
+Personality=ppc64
```

Comment:
Use `#if __BYTE_ORDER == __BIG_ENDIAN` for the endianess check. See src/basic/architecture.h, it has checks for all architectures we support.